core-bundle/contao/library/Contao/Controller.php

```diff
@@ -590,6 +588,9 @@ public static function getContentElement($intId, $strColumn='main')
 		}
 		else
 		{
+			$objRow = $objRow->cloneDetached();
+			$objRow->typePrefix = 'ce_';
```

Comment:
This code would still get executed for `ContentProxy` elements so we would need to add an extra check here?
```suggestion
			if (!$isContentProxy)
			{
				$objRow = $objRow->cloneDetached();
				$objRow->typePrefix = 'ce_';
			}
```
After this change, ESI worked again in my test.

But are we 100% sure that we don’t need the `typePrefix` for fragment controller content elements?